README.md

```diff
@@ -89,6 +89,46 @@ $ ansible-playbook --ask-vault-pass deploy_todoApp.yaml -i inventory
 7. In your browser enter the managed node IP address at port 5000 to view the app i.e.:
 123.123.123.123:5000
 
+## Dockerfile Image generation
+
+1. Install Docker:
+https://www.docker.com/products/docker-desktop/
+
+2. Install GUnicorn:
+```bash
+$ pip install gunicorn
+```
+
+3. Install Watchdog
+```bash
+$ poetry add watchdog --dev
+```
```

Comment:
No, you don't need to install gunicorn on your machine

And you never need to tell people to run `poetry add` anything. Your dependencies are recorded in `pyproject.toml`.

Reply:
Noted and edited. Thanks.